extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -357,4 +360,13 @@ public static function onLinkEnd( $skin, Title $target, array $options, &$text,
 
 		return true;
 	}
+
+	public static function onDisplayRedirectedFrom( $rdfrom, Article $article): bool {
+		if ( !$article->getContext()->getUser()->isAnon() ) {
+			$redir = Linker::makeExternalLink( $rdfrom, $rdfrom );
+			$article->getContext()->getOutput()->addSubtitle( wfMessage( 'redirectedfrom' )->rawParams( $redir ) );
```

Comment:
Minor: I'd suggest putting `$article->getContext()` in a variable to be used here and in the `if`, and you can then also make use of it in place of `wfMessage` with `$context->msg( 'redirectedfrom' )->rawParams( $redir )`.